mmif/utils/textdoc_helper.py

```diff
@@ -0,0 +1,9 @@
+import mmif
+
+
+def slice_text(mmif_obj, start: int, end: int) -> str:
+    sort_token_anns = mmif_obj.get_annotations_between_time(start, end)
+    tokens_sliced = []
+    for ann in sort_token_anns:
+        tokens_sliced.append(ann.get_property('text'))  # FIXME: Sometimes the string attribute "word" is used for getting property value
```

Comment:
maybe check if the `ann` is indead a `Token` type?

Reply:
* I still hardcoded a variable `token_type` with its defined url since I didn't see an attribute to access in codebase like `AnnotationTypes.<ThingBaseType>`. 
* Also, in order to access the string of `Token`, I found the property's name is `text` in test examples (run by kaldi: `http://mmif.clams.ai/apps/kaldi/0.2.1`) but is `word` in the output mmif run by whisper-wrapper.